letsencrypt_apache/display_ops.py

```diff
@@ -0,0 +1,76 @@
+"""Contains UI methods for Apache operations."""
+import os
+import zope.component
+
+from letsencrypt import interfaces
+
+import letsencrypt.display.util as display_util
+
+
+def select_vhost(domain, vhosts):
+    """Select an appropriate Apache Vhost.
+
+    :param vhosts: Available Apache Virtual Hosts
+    :type vhosts: :class:`list` of type `~obj.Vhost`
+
+    :returns: VirtualHost
+    :rtype: `~obj.Vhost`
+
+    """
+    if not vhosts:
+        return None
+    while True:
+        code, tag = _vhost_menu(domain, vhosts)
+        if code == display_util.HELP:
+            _more_info_vhost(vhosts[tag])
+        elif code == display_util.OK:
+            return vhosts[tag]
+        else:
+            return None
+
+
+def _vhost_menu(domain, vhosts):
+    """Select an appropriate Apache Vhost.
+
+    :param vhosts: Available Apache Virtual Hosts
+    :type vhosts: :class:`list` of type `~obj.Vhost`
+
+    :returns: Display tuple - ('code', tag')
+    :rtype: `tuple`
+
+    """
+    filename_size = 24
+    disp_name_size = 17
```

Comment:
They seemed to format the output nicely on both the terminal and ncurses.  That being said, it would be better if they were a function of the display size.